src/utils/vm_image_vault_utils.cpp

```diff
@@ -77,3 +78,19 @@ QString mp::vault::extract_image(const mp::Path& image_path, const mp::ProgressM
 
     return new_image_path;
 }
+
+std::unordered_map<std::string, mp::VMImageHost*>
+mp::vault::configure_image_host_map(std::vector<mp::VMImageHost*> image_hosts)
```

Comment:
Better avoid copy:

```suggestion
mp::vault::configure_image_host_map(const std::vector<mp::VMImageHost*>& image_hosts)
```

Reply:
Right, should've seen that before.  Thanks!